CompetitionTask1/CompetitionTask1/Pages/ManageListingsPage.cs

```diff
@@ -39,31 +39,31 @@ public class ManageListingsPage
         [FindsBy(How = How.XPath, Using = "//h3[contains(text(),'You do not have any service listings!')]")]
         public IWebElement deletedListing { get; set; }
 
-        public void ViewListings(IWebDriver driver)
+        public void ViewListings()
         {
             CommonDriver.UseWait();
             PageFactory.InitElements(driver, this);
             viewListingsbutton.Click();
             CommonDriver.UseWait();
-            Assert.That(listedTitle.Text == "ISTQB", "Expected and actual Title doesnot match");
+            //Assert.That(listedTitle.Text == "ISTQB", "Expected and actual Title doesnot match");
         }
 
-        public void GoToShareSkillPage(IWebDriver driver)
+        public void GoToShareSkillPage()
         {
             CommonDriver.UseWait();
             PageFactory.InitElements(driver, this);
             editButton.Click();
             CommonDriver.UseWait();
         }
         
-         public void DeleteShareSkill(IWebDriver driver)
+         public void DeleteShareSkill()
          {
                 CommonDriver.UseWait();
                 PageFactory.InitElements(driver, this);
                 deleteButton.Click();
                 deleteConfirm.Click();
                 CommonDriver.UseWait();
-                Assert.That(deletedListing.Text == "You do not have any service listings!", "Record  not deleted");
+                //Assert.That(deletedListing.Text == "You do not have any service listings!", "Record  not deleted");
```

Comment:
remove if no needed should never have commented code, also i think you forgot to add the asserts into the test, you should move this in the test class